### PR TITLE
Work session of the Portfolio setup - Mobile version

### PR DESCRIPTION
**Pull request by gids-dadi**
In the feature-work session branch,

-I added the work session to the portfolio setup project.

-You get to see previous work I have done and the technology used to create the projects.

 -Gallery to check out some of the previous projects

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+            <img src="icons_images/Vector5.png" alt="" />
+          </a>
+        </li>
+      </ul>
+
+      <div class="work-arrow">
+        <a href="#"> <img src="icons_images/Vector6.png" alt="" /> Work </a>
+      </div>
+
+      <h1 class="my-work-header">MY RECENT WORK</h1>
```
Comment (BenMukebo) [inline index.html]: - [x] Please change this h1 to hx grade below, as for HTML best practice, your web app should only have one h1 tag, consider doing this for all similar titles.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+          </p>
+
+          <div class="tech-used-1">
+            css <img src="icons_images/line.png" alt="Line" /> html
+            <img src="icons_images/line.png" alt="Line" />bootstrap
+            <img src="icons_images/line.png" alt="Line" />Ruby
+          </div>
+
+          <button class="button" type="button">See Project</button>
+        </div>
```
Comment (BenMukebo) [inline index.html:145]: - [x] great work up to here, your design looks awesome, but I think we can still make this better by adding some button action as per this image below, let's go and add that too 😄 , consider making the change on all similar buttons, please

hover

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@gids-dadi,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💪🏻

- No linter errors. ✔️ 
- Good commit messages ✔️ 
- Professional README ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [x] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (gids-dadi) [reply index.html]: Awesome! Thanks for the correction

**Review by Deepakdanger — APPROVED** [review]
Hi @gids-dadi, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**